src/rules.js

```diff
@@ -1903,6 +1903,80 @@ module.exports = [
     },
     /**
      ==================================================================
+     TRANSITION
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        id: 'transition',
+        name: 'Transition',
+        prefix: '.Trs-',
+        properties: ['transition'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true
+    },
+    {
+        type: 'pattern',
+        id: 'transition-delay',
+        name: 'Transition delay',
+        prefix: '.Trsde-',
+        properties: ['transition-delay'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'i', values: ['initial']}
+        ]
+    },
+    {
+        type: 'pattern',
+        id: 'transition-duration',
+        name: 'Transition duration',
+        prefix: '.Trsdu-',
+        properties: ['transition-duration'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'inh', values: ['inherit']}
+        ]
+    },
+    {
+        type: 'pattern',
+        id: 'transition-property',
+        name: 'Transition property',
+        prefix: '.Trsp-',
+        properties: ['transition-property'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'a', values: ['all']}
+        ]
+    },
+    {
+        type: 'pattern',
+        id: 'transition-timing-function',
+        name: 'Transition timing function',
+        prefix: '.Trstf-',
+        properties: ['transition-timing-function'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
```

Comment:
I don't understand why we'd want to support that - we already have rules defining `Trstf-e` and `Trstf-l` for ease and linear, and `Trstf-eio` for ease-in-out, so there's no reason to support suffix-to-value here.  Suffix-to-value should be reserved more for arbitrary measurement style values, I'd think.